### PR TITLE
CommunityScripts add-on

### PR DESCRIPTION
**Pull request by psiinon**

### CODE REVIEW

File: src/org/zaproxy/zap/extension/communityScripts/CommunityScriptFiles.java [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+import java.util.Comparator;
+
+/**
+ * Simple utility class for listing the files to include in the ZapAddOn.xml file
+ * @author simon
+ *
+ */
+public class CommunityScriptFiles {
+	
+	public static final String ORIG = "src/org/zaproxy/zap/extension/svndigger/orig/";
```
Comment (kingthorin) [inline src/org/zaproxy/zap/extension/communityScripts/CommunityScriptFiles.java]: ORIG is never used, that I could see. Plus it points to the directory of another extension.

Comment (psiinon) [inline src/org/zaproxy/zap/extension/communityScripts/CommunityScriptFiles.java]: What do you suggest? Removing the code?

**Comment by kingthorin** [comment]
Ya if it really isn't used, it should be removed. If it is used then just let me know what I missed.

File: src/org/zaproxy/zap/extension/communityScripts/CommunityScriptFiles.java (hunk 1 above)
Comment (kingthorin) [inline src/org/zaproxy/zap/extension/communityScripts/CommunityScriptFiles.java]: Ya if it really isn't used, it should be removed. If it is used then just let me know what I missed.

Comment (psiinon) [inline src/org/zaproxy/zap/extension/communityScripts/CommunityScriptFiles.java]: Removed :)

**Pull request merged**